packages/machine/src/instruction-executor.ts

```diff
@@ -1,11 +1,11 @@
 import * as cf from "@counterfactual/cf.js";
+import { Node } from "@counterfactual/node";
```

Comment:
This still seems like a massive antipattern to me. The `Node` should not be consumed by the `machine`. We should mark it as TO-FIX or something or just find a way to not do this in the first place.

Reply:
Not gonna mark as `FIXME` - rather not do it this way in the first place and introduce it to the codebase.